model/load_balancer.rb

```diff
@@ -0,0 +1,34 @@
+#  frozen_string_literal: true
+
+require_relative "../model"
+
+class LoadBalancer < Sequel::Model
+  many_to_many :vms
+  one_to_one :strand, key: :id
+  many_to_one :private_subnet
+  one_to_many :projects, through: :private_subnet
+
+  include ResourceMethods
+  include SemaphoreMethods
+  include Authorization::TaggableMethods
+  include Authorization::HyperTagMethods
+  semaphore :destroy, :update_load_balancer
+
+  def hyper_tag_name(project)
+    "project/#{project.ubid}/load-balancer/#{name}"
```

Comment:
Just to clarify, are load balancers location-specific or global resources?

Reply:
Subnets are location-specific, and load balancer seem to be children of subnets. It seems reasonable to make them location-specific. But their table doesn't have a location column.

Reply:
started using private subnet's location 👍